packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js

```diff
@@ -49,14 +74,47 @@ export default class RegionsListController {
     );
   }
 
-  updateRegions() {
-    const formattedRegions = map(this.regions, (region) => ({
-      ...this.ovhManagerRegionService.getRegion(region.name),
-      name: region.name,
-      continentCode: region.continentCode,
-      hasEnoughQuota: region.hasEnoughQuota(),
-      isLocalZone: region.isLocalZone,
-    }));
+  async updateRegions() {
+    if (this.deploymentMode) {
+      await this.PciProjectStorageContainersService.getProductAvailability(
+        this.projectId,
+        this.coreConfig.getUser().ovhSubsidiary,
+      ).then((productCapabilities) => {
+        const productCapability = productCapabilities.plans
+          ?.filter((plan) =>
+            plan.code?.startsWith(STORAGE_STANDARD_REGION_PLANCODE),
+          )
+          .filter(
+            (plan, index, self) =>
+              index === self.findIndex((p) => p.name === plan.name),
+          );
```

Comment:
Why do you separate the filter function in 2 parts ? It could be done once

Reply:
when i combine them it filters the duplicates too early and i get an empty result